pkg/console/operator/sync_v400.go

```diff
@@ -152,13 +152,7 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 		co.ConditionDeploymentNotAvailable(updatedOperatorConfig, msg)
 	} else if !routesub.IsAdmitted(rt) {
 		klog.V(4).Infoln("console route is not admitted")
-		co.SetStatusCondition(
-			updatedOperatorConfig,
-			operatorv1.OperatorStatusTypeAvailable,
-			operatorv1.ConditionFalse,
-			"RouteNotAdmitted",
-			"console route is not admitted",
-		)
+		co.HandleAvailable(updatedOperatorConfig, "RouteNotAdmitted", errors.New("console route is not admitted"))
```

Comment:
nit, lets not repeat ourselves
```suggestion
                msg := "console route is not admitted"
                klog.V(4).Infoln(msg)
		co.HandleAvailable(updatedOperatorConfig, "RouteNotAdmitted", errors.New(msg))
```